docs/api-overview.md

```diff
@@ -121,6 +133,7 @@ _Appears in:_
 | `api` _string_ |  |  |  |
 | `provider_id` _string_ |  |  |  |
 | `provider_type` _string_ |  |  |  |
+| `health` _[ProviderHealthStatus](#providerhealthstatus)_ |  |  |  |
```

Comment:
I think this should also include `config`

Reply:
Actually no, it's our `sed` command that removes link to doc? @VaishnaviHire why are we doing this?